ert/storage/_storage.py

```diff
@@ -46,13 +47,187 @@ class Config:
     record_type: ert.data.RecordType
 
 
-def _assert_server_info() -> None:
-    global _STORAGE_URL, _STORAGE_TOKEN  # pylint: disable=global-statement
+class StorageInfo:
+    _url: Optional[str] = None
+    _token: Optional[str] = None
 
-    if _STORAGE_URL is None:
+    @classmethod
+    def _set_info(cls) -> None:
         info = get_info()
-        _STORAGE_URL = info["baseurl"]
-        _STORAGE_TOKEN = info["auth"][1]
+        cls._url = info["baseurl"]
+        cls._token = info["auth"][1]
+
+    @classmethod
+    def url(cls) -> str:
+        if StorageInfo._url is None:
+            cls._set_info()
+        return str(StorageInfo._url)
+
+    @classmethod
+    def token(cls) -> str:
+        if StorageInfo._token is None:
+            cls._set_info()
+        return str(StorageInfo._token)
+
+
+class StorageRecordTransmitter(ert.data.RecordTransmitter):
+    _TYPE: ert.data.RecordTransmitterType = ert.data.RecordTransmitterType.ert_storage
+
+    def __init__(self, name: str, storage_path: str, iens: Optional[int] = None):
```

Comment:
Maybe call this `storage_url` right away? 
```suggestion
    def __init__(self, name: str, storage_url: str, iens: Optional[int] = None):
```